packages/xo-web/src/xo-app/vm/tab-advanced.js

```diff
@@ -1030,7 +1030,7 @@ export default class TabAdvanced extends Component {
                         onChange={this._handleBootFirmware}
                         value={defined(() => vm.boot.firmware, '')}
                       />
-                      {!vm.isFirmwareSupported && (
+                      {vm.boot.firmware !== 'uefi' && !vm.isFirmwareSupported && (
```

Comment:
Maybe leave a comment for this because, as I understand, the templates are wrongly reporting that UEFI is not supported so this is just a workaround. Right?